packages/common/src/data/location_data.json

```diff
@@ -1,9 +1,9 @@
 {
   "has": {
     "byId": {
-      "VancouverRegion": {
-        "id": "VancouverRegion",
-        "name": "Vancouver Region",
+      "VancouverCostal": {
```

Comment:
Can we make sure to update the source data for this and rerun the script to verify the output?